v2/pcb-v2.1/rgb-led-v2.kicad_pcb

```diff
@@ -146,11 +146,11 @@
     (fp_line (start 28.2 0) (end 28.2 10.5) (layer F.SilkS) (width 0.15))
     (fp_line (start 0 0) (end 28.2 0) (layer F.SilkS) (width 0.15))
     (fp_line (start 0 10.5) (end 0 0) (layer F.SilkS) (width 0.15))
-    (pad 1 thru_hole oval (at 3.4 5.25 90) (size 2 8.72) (drill oval 1 7.22) (layers *.Cu *.Mask F.SilkS)
+    (pad 1 thru_hole oval (at 3.4 5.25 90) (size 2.2 8.8) (drill oval 1.1 7.3) (layers *.Cu *.Mask F.SilkS)
```

Comment:
1 -> 1.1 is probably OK (increase by 10%). The increase from 7.22 to 7.3 is probably not necessary, but won't hurt.